tests/unit/insertTestResult.unitTest.ts

```diff
@@ -1500,4 +1500,401 @@ describe("insertTestResult", () => {
                 });
         });
     });
+
+    // CVSB-7964: AC4
+    context("when inserting an PSV test result for LEC test code with mandatory fields: certificateNumber, expiryDate, modType, emissionStandard and fuelType populated for Pass tests", () => {
+        it("test record should get created and should not throw error", () => {
+            const testResult = testResultsPostMock[8];
```

Comment:
inherits mutated record from above. Might  suit the test, but relies on order of execution. Unsafe.

Reply:
cloneDeep used to avoid using mutated record.